src/audiomixerboard.cpp

```diff
@@ -807,13 +816,32 @@ void CAudioMixerBoard::ChangeFaderOrder ( const bool        bDoSort,
     }
 }
 
+QString CAudioMixerBoard::GetTitle()
```

Comment:
FYI: In my "merge fix" I renamed this to UpdateTitle(). The functionality should still be the same as yours.